spec/ScoreCardSpec.js

```diff
@@ -0,0 +1,159 @@
+describe("ScoreCard", function() {
+  var scorecard;
+
+  beforeEach(function() {
+    scorecard = new ScoreCard();
+  });
+
+  it("should keep track of the frame", function() {
```

Comment:
Avoid starting test specifications with modal verbs like 'should'. For me, the modality is implied by it being a test.